readthedocs/projects/tasks.py

```diff
@@ -966,7 +997,7 @@ def build_docs_localmedia(self):
 
     def build_docs_pdf(self):
         """Build PDF docs."""
-        if 'pdf' not in self.config.formats:
+        if 'pdf' not in self.config.formats or self.version.type == EXTERNAL:
```

Comment:
I think we should consider to build these formats on PRs at some point. It'd be pretty handy to test different configurations on PDFs --you can always activate a branch, though.

Reply:
I'm really worried about resource usage of doing this at a high scale, so we are trying to keep it as little as possible for now.